rasa/core/agent.py

```diff
@@ -271,6 +271,14 @@ def _load_and_set_updated_model(
         raise
 
 
+def _contains_unpacked_model(model_path: Text) -> bool:
+    try:
+        core_path, nlu_path = get_model_subdirectories(model_path)
```

Comment:
```suggestion
        _ = get_model_subdirectories(model_path)
```

Reply:
Good point haha, for some reason I was like I have to make it `_, _` and that'd be weird.